dissect/target/helpers/regutil.py

```diff
@@ -172,6 +172,18 @@ def value(self, value: str) -> RegistryValue:
         """
         raise NotImplementedError()
 
+    def value_or_default(self, value: str, default: ValueType = None) -> RegistryValue:
```

Comment:
Can you roll this into the `value` method so it's more in line with standard library Python code?